lib/common/acl.c

```diff
@@ -205,22 +222,20 @@ __xml_acl_parse_entry(xmlNode *acl_top, xmlNode *acl_entry, GList *acls)
     </acls>
 */
 
-#ifdef SUSE_ACL_COMPAT
 static const char *
 __xml_acl_to_text(enum xml_private_flags flags)
 {
     if (is_set(flags, xpf_acl_deny)) {
         return "deny";
 
-    } else if (is_set(flags, xpf_acl_write)) {
+    } else if (is_set(flags, xpf_acl_write) || is_set(flags, xpf_acl_create)) {
```

Comment:
4'. I think I'd strictly separate internal state reporting from what it
    means semantically and would therefore avoid this joining

* it's a special internal only flag (it's presence alone indicates
  a delicate context of use), so a dedicated label would fit it better,
  IMHO

(counterproposal then would be to analyse whether it's really that
special and whether to join it right away perhaps -- likely not, there's
a special interaction with yet another `xpf_created` mode/state)

Reply:
xpf_acl_create gets remapped to xpf_acl_write in pcmk__check_acl(), and especially since it is internal only, I thought it made sense to report it from the point of view of what the user sees

Reply:
I see, I've meant the context you fire it up yourself in debugger, for instance.